charts/tfy-agent/values.yaml

```diff
@@ -1,123 +1,304 @@
-# Default values for sample.
+# Default values for tfy-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-global: {}
+# global settings
+global:
+  rbac:
+    enabled: true
 
-image:
-  repository: truefoundrycloud/tfy-agent
-  pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "a9af8a51d8b0373f4d0c469119ed88a488c5709f"
+# configuration for tfy-agent and tfy-agent-proxy
+config:
+  # token to connect to the contorl plane
+  clusterToken: ""
+
+  # name of the secret containing the cluster token
+  # the key must be token
+  # if this is set it will override config.clusterToken
+  clusterTokenSecret: ""
```

Comment:
if user provides a `clusterTokenSecret` the expectation might be that it already exists in the cluster. But current behaviour seems to be to create a new secret even when `clusterTokenSecret` is provided.